core/src/main/java/org/camunda/connect/Connectors.java

```diff
@@ -34,13 +34,17 @@
  */
 public class Connectors {
 
-  protected static ConnectCoreLogger LOG = ConnectLogger.CORE_LOGGER;
+  protected static final ConnectCoreLogger LOG = ConnectLogger.CORE_LOGGER;
```

Comment:
```suggestion
  protected static ConnectCoreLogger LOG = ConnectLogger.CORE_LOGGER;
```

❌  Please don't change public API classes in a way that is not backward compatible. You can find out about backward incompatibilities with the output of the Maven Clirr plugin.

Reply:
As stated in my original PR, I am aware of these _breaking_ changes clirr is complaining about. However in my opinion this is no public API to begin with. It is common practise to define a logger to be `private static final`, hence I aimed to improve code quality by adding the `final` keyword as I can only imagine it was forgotten. In this instance there may be reasoning to define it `protected` however I can not imagine a single use case where one - rightfully - would need to manipulate this field.